lua/efmls-configs/formatters/black.lua

```diff
@@ -5,9 +5,14 @@
 local fs = require('efmls-configs.fs')
 
 local formatter = 'black'
-local command = string.format('%s --no-color -q -', fs.executable(formatter))
+
+local command = string.format(
+  "%s --no-color -q $(echo ${--useless:rowStart} ${--useless:rowEnd} | xargs -n4 -r sh -c 'echo --line-ranges=$(($1+1))-$(($3+1))') -",
```

Comment:
This doesn't seem to be cross-platform supported, is there any way to not use `xargs`?

Reply:
> This doesn't seem to be cross-platform supported, is there any way to not use `xargs`?

Black has only one option for settings `rowStart` and `rowEnd`. Here's the solution I found: https://github.com/mattn/efm-langserver/issues/251
I don't know what would be the alternative on Mac OS and Windows as I don't know use them

Reply:
Did some digging on the efm-langserver codebase and doesn't seem like there is a way to de-couple `rowStart` and `rowEnd` from flags it's passed on to.

I think we can separate the format commands for now. Keep the old one if on windows, and use the new one if on linux/mac.

I'll approve this for now and add the changes later, thanks.